src-electron/db/zap-schema.sql

```diff
@@ -808,6 +808,7 @@ CREATE TABLE IF NOT EXISTS "ENDPOINT" (
   "PROFILE" integer,
   "ENDPOINT_IDENTIFIER" integer,
   "NETWORK_IDENTIFIER" integer,
+  "PARENT" integer,
```

Comment:
This should be called `PARENT_ENDPOINT_REF`.

Reply:
yes will do